webdiff/app.py

```diff
@@ -51,6 +52,7 @@ class Config:
 
 DIFF = None
 PORT = None
+HOSTNAME = "localhost"
```

Comment:
Please use single quotes everywhere. (It looks like this file uses a mix but I generally stick with consistent single quotes.)